Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -80,36 +95,34 @@ const CreateNewMaintenanceWindow = () => {
   const handleSubmit = async () => {
     const data = {
       repeat: values.repeat,
-      date: values.date.format("YYYY-MM-DD"),
-      startTime: values.startTime.format("HH:mm"),
+      date: values.date.utc().format("YYYY-MM-DD"),
+      startTime: values.startTime.utc().format("HH:mm"),
```

Comment:
We use typical unix timestamps measured in ms throughout the app, both on the frontend and the backend, so the date needs to be in that format.  Conversion to human readable date/time will be done in the components that use the timestamp.

We don't need to submit the date and start times seperately, just the timestamp.

Duration should also be converted to ms

TLDR we need a timestamp, you [can get it from DayJS](https://day.js.org/docs/en/display/unix-timestamp-milliseconds)

Reply:
almost fixed ✅❗
There are some parts that could not be deleted, since removing them makes other parts like validations broken